config/example_options.ini

```diff
@@ -166,6 +166,9 @@ SearchList = no
 # Set the amount of search results returned by default.
 DefaultSearchResults = 3
 
+# Allow per-guild command prefix override via 'setprefix' command.
+EnableOptionsPerGuild = no
```

Comment:
Personally I'd think something like `EnablePrefixPerGuild` would be better.

Reply:
I agree.  I started this intending to add more options but ended up just doing command prefix to keep it focused.